### DISCUSSION TITLE
Chia node synchro stop yesterday block 5,496,001 - under Chia version 1.51

### DISCUSSION THREAD

**Discussion by Leixxington** [Chia-Network/chia-blockchain#18179] [Support]
I would like to update to the latest version but I do not know how to do it without synchro the DB from the start.
Running under Ubuntu 22.
Does anyone have the code to update from terminal ?
Thx

**Comment by BrandtH22** [comment]
Hey @Leixxington , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

To upgrade from version 1.5.1 you will need to:
- uninstall chia
- install the latest version of chia
- upgrade your v1 db to a v2 db (there are two methods for this based on how much storage you have available, both are listed here https://docs.chia.net/faq/#how-do-i-upgrade-my-version-1-blockchain-database-to-version-2)
- have a new config file created (rename the config file at `~/.chia/mainnet/config/config.yaml` then run `chia init` to have a new one created, you will need to update the config with any custom settings (plot directories, trusted peer info, etc)
- if you have nft plots I would also resync the wallet (this ensures the new config file has the correct pool information). There is a GUI option in settings / advanced for resync or you can stop chia, rename the wallet db at `~/.chia/mainnet/wallet/db/`, and start chia to force a reesync
- once all that is done monitor your farm and verify everything is working as intended

Here are the install instructions for cli:
https://docs.chia.net/installation/#using-the-cli

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
